### ISSUE TITLE
Parallel queue?

### ISSUE THREAD

**Issue by MatiSanchez** [jamesmh/coravel#279]
Hi!! 

Is it possible to run queue in parallel like schedules with `OnWorker`?

I'm using several `QueueInvocableWithPayload` and the problem is that you have to wait for one to finish before the other is executed.

I searched in docs but didn't find any info on this.

**Comment by jamesmh** [comment]
Yes, this is not available at the moment. It is a feature that's on the radar though, but no def. timeframe at the moment. Will update this thread when progress is made 👍

**Comment by jamesmh** [comment]
#139
